js/client.go

```diff
@@ -56,6 +56,11 @@ func (c *Client) JSValue() js.Value {
 		"addDACPolicy":               goji.Async(c.addDACPolicy),
 		"addDACActorRelationship":    goji.Async(c.addDACActorRelationship),
 		"deleteDACActorRelationship": goji.Async(c.deleteDACActorRelationship),
+		"getAACStatus":               goji.Async(c.getAACStatus),
+		"reEnableAAC":                goji.Async(c.reEnableAAC),
+		"disableAAC":                 goji.Async(c.disableAAC),
+		"addAACActorRelationship":    goji.Async(c.addAACActorRelationship),
+		"deleteAACActorRelationship": goji.Async(c.deleteAACActorRelationship),
```

Comment:
todo: A reminder to add those to the txn as well.

Reply:
There is not a way currently to easily catch this as the js tests don't use the newly available transaction object. I'm not sure its worth changing that for now though.